[self-tests] test_py_info_machine_property fails on i686 platform

`test_py_info_machine_property` fails on `i686` platform (expected `x86`):
```
________________________ test_py_info_machine_property _________________________

    def test_py_info_machine_property() -> None:
        machine = CURRENT.machine
        assert machine is not None
        assert isinstance(machine, str)
        assert len(machine) > 0
        known_isas = {"arm64", "x86_64", "x86", "ppc64le", "ppc64", "s390x", "riscv64"}
>       assert machine in known_isas, f"unexpected machine value: {machine}"
E       AssertionError: unexpected machine value: i686
E       assert 'i686' in {'arm64', 'ppc64', 'ppc64le', 'riscv64', 's390x', 'x86', ...}

tests/py_info/test_py_info.py:342: AssertionError
```

--- a/tests/py_info/test_py_info.py
+++ b/tests/py_info/test_py_info.py
@@ -338,7 +338,7 @@ def test_py_info_machine_property() -> None:
     assert machine is not None
     assert isinstance(machine, str)
     assert len(machine) > 0
-    known_isas = {"arm64", "x86_64", "x86", "ppc64le", "ppc64", "s390x", "riscv64"}
+    known_isas = {"arm64", "i686", "ppc64", "ppc64le", "riscv64", "s390x", "x86", "x86_64"}
     assert machine in known_isas, f"unexpected machine value: {machine}"
 
 

